### PR TITLE
find person service

### PR DESCRIPTION
**Pull request by udaydenduluri33**
https://tools.hmcts.net/jira/browse/EUI-4070

Find person Component display changes for Judicial user

### CODE REVIEW

**Review by rayliangatos — APPROVED** [review]

File: projects/exui-common-lib/src/lib/components/find-person/find-person.component.ts [hunk 1]
```diff
@@ -61,6 +61,9 @@ export class FindPersonComponent implements OnInit {
   }
 
   public getDisplayName(selectedPerson: Person) {
```
Comment (kuda1992) [inline projects/exui-common-lib/src/lib/components/find-person/find-person.component.ts:63]: return type missing

**Review by kuda1992 — CHANGES_REQUESTED** [review]
Very minor change requested but all good 😌

File: projects/exui-common-lib/src/lib/components/find-person/find-person.component.ts (hunk 1 above)
Comment (udaydenduluri33) [reply projects/exui-common-lib/src/lib/components/find-person/find-person.component.ts]: Done

**Review by kuda1992 — APPROVED** [review]
Thanks for making the change @udaydenduluri33

**Pull request merged**